checks/http/plesk.rb

```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: "fingerprint",
               category: "service",
-              tags: ["COTS"],
+              tags: ["SaaS"],
```

Comment:
is plesk just SaaS nowadays? I feel like it always had an on-premise installation option